spec/support/lazy_load_helpers.cr

```diff
@@ -1,4 +1,5 @@
 module LazyLoadHelpers
+  # ameba:disable Style/RedundantBegin
```

Comment:
Could this be changed to just `ensure` without the `begin` so we don't need to disable the rule?